otwcode/otwarchive
# AO3-5809 Add paste to RTE context menu and toolbar, and and hr to the context menu

##  Issue

https://otwarchive.atlassian.net/browse/AO3-5809

## Purpose

Adds paste and hr options to the RTE's context menu.

## Testing Instructions

On desktop, go to the RTE. Right-click. Confirm options are there and do relatively reasonable things.

## References

https://www.tiny.cloud/docs/configure/editor-appearance/#contextmenu
